scripts/variantstore/beta_docs/gvs-outputs.md

```diff
@@ -0,0 +1,126 @@
+##FILTER=<ID=EXCESS_ALLELES,Description="Site has an excess of alternate alleles based on the input threshold">
+##FORMAT=<ID=AD,Number=R,Type=Integer,Description="Allelic depths for the ref and alt alleles in the order listed">
+##FORMAT=<ID=FT,Number=1,Type=String,Description="Genotype Filter Field">
+##FORMAT=<ID=GQ,Number=1,Type=Integer,Description="Genotype Quality">
+##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
+##FORMAT=<ID=RGQ,Number=1,Type=Integer,Description="Unconditional reference genotype confidence, encoded as a phred quality -10*log10 p(genotype call is wrong)">
+##INFO=<ID=AC,Number=A,Type=Integer,Description="Allele count in genotypes, for each ALT allele, in the same order as listed">
+##INFO=<ID=AF,Number=A,Type=Float,Description="Allele Frequency, for each ALT allele, in the same order as listed">
+##INFO=<ID=AN,Number=1,Type=Integer,Description="Total number of alleles in called genotypes">
+##INFO=<ID=AS_QUALapprox,Number=1,Type=String,Description="Allele-specific QUAL approximations">
+##INFO=<ID=QUALapprox,Number=1,Type=Integer,Description="Sum of PL[0] values; used to approximate the QUAL score">
+##COMMENT=<ID=high_CALIBRATION_SENSITIVITY_SNP,Description="Sample Genotype FT filter value indicating that the genotyped allele failed SNP model calibration sensitivity cutoff (0.997)">
+##COMMENT=<ID=high_CALIBRATION_SENSITIVITY_INDEL,Description="Sample Genotype FT filter value indicating that the genotyped allele failed INDEL model calibration sensitivity cutoff (0.99)">
+```
+
+Exomes will additionally have:
+
+```
+##FILTER=<ID=OUTSIDE_OF_TARGETS,Description="Outside of sequencing target intervals">
+```
+
+## Notable content NOT in the VCF
+### DP
+DP is not in GVS VCFs. It can be approximated from SUM of AD and in most cases should be equal, unless there were some low AF alternates that were dropped in reblocking. AD is a 2 or 3 length vector (only 3 if ½ genotype) and is only included for variant sites.
+
+### PL: phred-scaled genotype likelihoods
+PLs (phred-scaled genotype likelihoods) are critically important for downstream analysis, but they can (mostly) be derived from the GQ value. Removing PLs from VCFs has enabled us to reduce file sizes by over an order of magnitude because the typical PL representation leads to an explosion of VCF sites at highly multi-alleleic sites.
+
+Note that normally the array for PL is length "G" where G is the number of genotypes. Instead, we output "." which is length 1, to represent the missing PL format field. 
+
+PL can be approximated as [0, GQ, ~2*GQ]
+
+### Missing (`./.`) and `GQ0 0/0`
+Reblocking, a step required before running GVS, combines `./.` and `0/0`, see quote from reblocking release notes:
+> Missing (`./.`) genotype calls will be output at `GQ0` homozygous reference calls in the new pipeline version, so QC methods that rely on “missingness” will also need to be modified.
+
+On output to both VCF and VDS, we output `./.` and `0/0 GQ0` calls as `./.`. 
```

Comment:
You said GVS Beta is VCF-only, right?  So maybe that bit about about the "new pipeline version" isn't important?

Reply:
I can see how this is confusing. The part that says "new pipeline version" is referring to the reblocking pipeline version because that's a direct quote from the reblocking documentation. I can [sic] edit the reblocking doc quote.

Reply:
Updated this section, can you let me know if it's clearer now?